n_queen_problem/n_queen_problem.go

```diff
@@ -0,0 +1,69 @@
+// Given a chess board having N×N cells
+// place N queens on the board in such a way that no queen attacks any other queen.
+
+// There can be a number of possible solutions for a given length of board.
+// This implementation prints only one valid solution, it can be extended to print all possible valid solutions.
+
+package main
+
+import "fmt"
+
+func main() {
+	var n int
+	print("Enter the value of 'N' ")
+	fmt.Scanf("%d", &n)
```

Comment:
No need to ask the value of `n`, take any sample value.

Reply:
Ok I'll set it to 8